android/src/main/java/com/dooboolab/RNIap/RNIapModule.java

```diff
@@ -158,7 +158,7 @@ public void initConnection(final Promise promise) {
     ensureConnection(promise, new Runnable() {
       @Override
       public void run() {
-        promise.resolve(true);
+        promise.resolve("true");
```

Comment:
Could you change the string into something readable? Something like `android billing connected`?

Reply:
I used `"true"` because iOS returns the same. Are you sure about this?

Reply:
Oh. Then it would be better for ios to give better string too.  Something like `Can make payment in ios.`. Since `true` is very useless.

Reply:
Since the promise will only be resolved if `ensureConnection` succeeds and there is only one outcome. It think, it would be better to return a Boolean `true` here, and in iOS also. Or just make the function void. This would make it easier to consume the outcome in JS, because it's much easier to check for Boolean than to compare a string. A string message would make sense if there were multiple successful outcomes.